app/views/Project/EntryEdit/index.tsx

```diff
@@ -589,7 +591,9 @@ function EntryEdit(props: Props) {
         [handleSubmit],
     );
 
-    const [selectedEntry, setSelectedEntry] = useState<string | undefined>();
+    const [selectedEntry, setSelectedEntry] = useState<string | undefined>(
+        entryIdFromState ?? undefined,
+    );
```

Comment:
Due to this new UI, if a entry is selected all of the navigations will be blocked.
Only way to resume navigation is to either accept or reject the changes.

There would be a problem if we set an entry as selected that doesn't exist.
So instead of setting selectedEntry here, we should can check if the selectedEntry is in the list of entries that we get from the server